x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -400,7 +400,14 @@ processors:
      if: "ctx._temp_.cisco.message_id == '338301'"
      field: "server.port"
      value: "{{source.port}}"
-
+ - dissect:
+     if: "ctx._temp_.cisco.message_id == '734001'"
+     field: "message"
+     pattern: "DAP: User %{user.email}, Addr %{source.ip}, Connection %{cisco.connection_type}: The following DAP records were selected for this connection: %{cisco.dap_records->}"
```

Comment:
A few issues with this pattern:
1. Should set `source.address` instead of `source.ip`. A later processor will set `.ip` from `.address` if it's a valid IP.
2. Should set `_temp_.cisco.xxx` instead of `cisco.xxx` fields. A later processor creates `cisco.asa.xxx` or cisco.ftd.xxx` based on `_temp_.cisco`.